halo2_proofs/src/dev/graph/layout.rs

```diff
@@ -82,7 +82,7 @@ impl CircuitLayout {
     }
 
     /// Renders the given circuit on the given drawing area.
-    pub fn render<F: Field, ConcreteCircuit: Circuit<F>, DB: DrawingBackend>(
+    pub fn render<F: PrimeField, ConcreteCircuit: Circuit<F>, DB: DrawingBackend>(
```

Comment:
Ditto: why is this change necessary?

Reply:
I saw above. Nor sure neither. It doesn't seem that anything requires any function from the `PrimeField` trait specifically right?

Reply:
Please see discussion at https://github.com/zcash/halo2/pull/715#discussion_r1115669042.